volunteer/forms.py

```diff
@@ -8,6 +11,120 @@ class VolunteerProfileForm(ModelForm):
     class Meta:
         model = VolunteerProfile
         exclude = ["user", "application_status"]
+        help_texts = {
+            "github_username": "GitHub username (e.g., username)",
+            "discord_username": "Discord username (e.g., username)",
+            "instagram_username": "Instagram username without @ (e.g., username)",
+            "bluesky_username": "Bluesky username (e.g., username or username.bsky.social)",
+            "mastodon_url": "Mastodon handle (e.g., @username@instance.tld or https://instance.tld/@username)",
+            "x_username": "X/Twitter username without @ (e.g., username)",
+            "linkedin_url": "LinkedIn URL (e.g., linkedin.com/in/username)",
+        }
+
+    def clean_github_username(self):
+        github_username = self.cleaned_data.get("github_username")
+        if github_username:
+            if not re.match(
+                r"^[a-zA-Z0-9](?:[a-zA-Z0-9]|-(?=[a-zA-Z0-9])){0,38}$", github_username
+            ):
+                raise ValidationError(
+                    "GitHub username can only contain alphanumeric characters and hyphens, "
+                    "cannot start or end with a hyphen, and must be between 1-39 characters."
+                )
+        return github_username
+
+    def clean_discord_username(self):
+        discord_username = self.cleaned_data.get("discord_username")
+        if discord_username:
+            if not re.match(
+                r"^[a-zA-Z0-9](?:[a-zA-Z0-9]|[._-](?=[a-zA-Z0-9])){0,30}[a-zA-Z0-9]$",
+                discord_username,
+            ):
+                if len(discord_username) < 2 or len(discord_username) > 32:
+                    raise ValidationError(
+                        "Discord username must be between 2 and 32 characters."
+                    )
+                else:
+                    raise ValidationError(
+                        "Discord username must consist of alphanumeric characters, "
+                        "dots, underscores, or hyphens, and cannot have consecutive special characters."
+                    )
+        return discord_username
+
+    def clean_instagram_username(self):
+        instagram_username = self.cleaned_data.get("instagram_username")
+        if instagram_username:
+            if instagram_username.startswith("@"):
+                instagram_username = instagram_username[1:]
+
+            if not re.match(r"^[a-zA-Z0-9._]{1,30}$", instagram_username):
+                raise ValidationError(
+                    "Instagram username can only contain alphanumeric characters, "
+                    "periods, and underscores, and must be between 1-30 characters."
+                )
+        return instagram_username
+
+    def clean_bluesky_username(self):
+        bluesky_username = self.cleaned_data.get("bluesky_username")
+        if bluesky_username:
+            if bluesky_username.startswith("@"):
```

Comment:
Same here, I think it should result in validation error instead of silently removing the @ from the username.